.github/workflows/tests.yml

```diff
@@ -46,4 +39,4 @@ jobs:
         run: python -m pip install '.[test]'
 
       - name: Run basic tests
-        run: python -m pytest -m "not cern_network" --mpl --cov-report xml --cov=pylhc
+        run: python -m pytest -m "not cern_network" --mpl
```

Comment:
Why the removal, are these default options now?

Reply:
why were they there in the first place?

Reply:
Aah this is not the coverage workflow, my bad! All good